Warning, the following versions were found for Mezzanine
3.1.10
3.1.9
3.1.8
3.1.7
3.1.6
3.1.5
3.1.4
3.1.3
3.1.2
3.1.1
3.1.0
3.0.9
3.0.8
3.0.7
3.0.6
3.0.5
3.0.4
3.0.3
3.0.2
3.0.1
3.0.0
1.4.16
1.4.15
1.4.14
1.4.13
1.4.12
1.4.11
1.4.10
1.4.9
1.4.8
1.4.7
1.4.6
1.4.5
1.4.4
1.4.3
1.4.2
1.4.1
1.4.0
1.3.0
1.2.4
1.2.3
1.2.2
1.2.1
1.2.0
1.1.4
1.1.3
1.1.2
1.1.1
1.1.0
1.0.10
1.0.9
1.0.8
1.0.7
1.0.6
1.0.5
1.0.4
1.0.3
1.0.2
1.0.1
1.0.0
0.12.5
0.12.4
0.12.3
0.12.2
0.12.1
0.12
0.11.10
0.11.9
0.11.8
0.11.7
0.11.6
0.11.5
0.11.4
0.11.3
0.11.2
0.11.1
0.11
0.10.6
0.10.5
0.10.4
0.10.3
0.10.2
0.10.1
0.10
0.9.1
0.9
0.8.5
0.8.4
0.8.3
0.8.2
0.8.1
0.8
0.7.2
0.7
0.6.4
0.6.3
0.6.2
0.6.1
0.6
0.5.4
0.5.3
0.5.2
0.5.1
0.4
0.3.5
0.3.4
0.3.3
0.3.2
0.3.1
0.3.0
0.2.4
0.2.3
0.2.2
0.2.1
0.2
0.1.4
0.1.3
0.1.2
0.1.1
0.1
Using 3.1.10
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/M/Mezzanine/Mezzanine-3.1.10.tar.gz (7.1 MB) for Mezzanine.
Downloading Mezzanine (use --no-download to skip this step)
Using cached download
Unpacking Mezzanine...
done
working in /tmp/tmpxfe33sconda_skeleton_Mezzanine-3.1.10.tar.gz
Fetching package metadata: ......
Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).
Traceback (most recent call last):
  File "setup.py", line 4, in <module>
    from setuptools import setup, find_packages
ImportError: No module named setuptools

Applying patch: '/tmp/tmpxfe33sconda_skeleton_Mezzanine-3.1.10.tar.gz/pypi-distutils.patch'
$PYTHONPATH = /tmp/tmpxfe33sconda_skeleton_Mezzanine-3.1.10.tar.gz/Mezzanine-3.1.10
Error: command failed: /home/ubuntu/miniconda/envs/_build/bin/python setup.py install
